README.md

```diff
@@ -13,9 +13,9 @@ SDK for generating and publishing events that capture user, application and tech
 
 <p align="center"><img src="https://static1.squarespace.com/static/5a123416bce176a964daebe5/t/5aa18123c83025fedf718a51/1554300899717/?format=1500w"></p>
```

Comment:
@robertdumitrescu Browser compatibility needs to be updated

Reply:
Will do it under another merge request as per our verbal conversation.